WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -7457,7 +7457,7 @@ which should be translated separately and considered part of this sentence. */
 "pointOfSale.cardPresent.preparingCardReader.cancel.button.title" = "Cancel";
 
 /* Message shown on the Point of Sale checkout while the reader is being prepared. */
-"pointOfSale.cardPresent.preparingCardReader.message" = "Preparing card reader";
+"pointOfSale.cardPresent.preparingCardReader.message" = "Checking order";
```

Comment:
nit: how about using a different key to replace the "preparingCardReader" part for this string, since the message doesn't seem to be about "preparing card reader" anymore? We wouldn't need to update the strings file anymore.

Reply:
I left it like that since it is in the "preparing card reader" stage, where the title is "Preparing card reader" and this is the message below the title.